src/main/java/com/devcycle/sdk/server/local/model/DVCLocalOptions.java

```diff
@@ -0,0 +1,39 @@
+package com.devcycle.sdk.server.local.model;
+
+import com.devcycle.sdk.server.common.model.IDVCOptions;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class DVCLocalOptions {
```

Comment:
It might make sense to move some of the options validation logic to a constructor here.